tea.js

```diff
@@ -0,0 +1,51 @@
+class Tea {
+    constructor(name, description) {
+        this.name = name;
+        this.description = description;
+    }
+}
+
+function displayTea(teaList) {
+    const teaContainer = document.getElementById('tea-container'); 
+    
+    teaList.forEach((teaItem, index) => {
+        const tea = new Tea(teaItem.name, teaItem.description);
+
+        const teaElement = document.createElement('div');
+        teaElement.classList.add('tea-item');
+
+        const teaName = document.createElement('h2');
+        teaName.textContent = tea.name;
+
+        const teaDescription = document.createElement('p');
+        teaDescription.textContent = tea.description;
+
+        teaElement.appendChild(teaName);
+        teaElement.appendChild(teaDescription);
```

Comment:
To make your code more organized, you can do something similar to how we did in last week's class example, where you move this block of code into the class `Tea` in a function called something like `display()`.